src/main/java/org/opentripplanner/framework/geometry/WgsCoordinate.java

```diff
@@ -29,43 +29,66 @@ public WgsCoordinate(double latitude, double longitude) {
     DoubleUtils.requireInRange(latitude, LAT_MIN, LAT_MAX, "latitude");
     DoubleUtils.requireInRange(longitude, LON_MIN, LON_MAX, "longitude");
 
-    // Normalize coordinates to precision around ~ 1 centimeters (7 decimals)
-    this.latitude = DoubleUtils.roundTo7Decimals(latitude);
-    this.longitude = DoubleUtils.roundTo7Decimals(longitude);
+    this.latitude = latitude;
+    this.longitude = longitude;
   }
 
   public WgsCoordinate(Point point) {
```

Comment:
I would prefer to make all constructors private to enforce the use of the factory methods, this is a minor thing and we could do it later in another PR if that is more convenient.

Reply:
Just curious, what is your reasoning for preferring factory methods in this case?

Reply:
I converted the `WgsCoordinate` into a record and kept only the canonical constructor, what do you think about that @t2gran ?

Reply:
As discussed in the dev-meeting, I do not want this as a record.
 - We have a policy of not using records for domain classes - this is an edge case - so might be ok...
 - For memory/performance reasons I would like to hide the fields lat/long - we might convert these to `int` at som point. 
  - Small gain, but medium risk. The `toString()` should be kept(value-object), the hc/eq are simple.